CMSIS/Core/Include/cmsis_gcc.h

```diff
@@ -132,22 +132,17 @@ __STATIC_FORCEINLINE __NO_RETURN void __cmsis_start(void)
 {
   extern void _start(void) __NO_RETURN;
   
-  typedef struct {
+  struct __copy_table_t{
     uint32_t const* src;
     uint32_t* dest;
     uint32_t  wlen;
-  } __copy_table_t;
+  } __copy_table_start__, __copy_table_end__;
```

Comment:
This defines new symbols in local scope. The code below relies on getting these symbols defined by the linker. Hence the symbols were declared as defined externally.

Reply:
I appreciate the clarification.